cluster/centos/config-default.sh

```diff
@@ -14,24 +14,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+readonly root=$(dirname "${BASH_SOURCE}")
+
 ## Contains configuration values for the CentOS cluster
 # The user should have sudo privilege
 export MASTER=${MASTER:-"centos@172.10.0.11"}
 export MASTER_IP=${MASTER#*@}
 
+# Define all your master nodes,
+# And separated with blank space like <user_1@ip_1> <user_2@ip_2> <user_3@ip_3>.
+# The user should have sudo privilege
+export MASTERS="${MASTERS:-$MASTER}"
+
+# length-of <arg0>
+# Get the length of specific arg0, could be a space-separate string or array.
+function length-of() {
+  local len=0
+  for part in $1; do
+    let len++
+  done
+  echo $len
+}
+# Number of nodes in your cluster.
+export NUM_MASTERS="${NUM_MASTERS:-$(length-of "$MASTERS")}"
+
+# Define advertise address of masters, could be a load balancer address.
+# If not provided, the default is ip of first master node.
```

Comment:
Can you rearrange the comment? It looks like a function comment but the function is not setting load balancer address.